services/mailservers/tcp_ping.go

```diff
@@ -0,0 +1,41 @@
+package mailservers
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/status-im/status-go/tcpping"
```

Comment:
I would call it `github.com/status-im/status-go/rtt`

Reply:
done